Addressbook/src/addressbook.sh

```diff
@@ -1,2 +1,62 @@
 #!/bin/ash
-echo "AddressBook"
\ No newline at end of file
+
+source module/insert-user.sh
```

Comment:
Nice modularization! I see some circular "imports" though, revealing that some functionality would be better moved into it's own "module". For example the functions that "access the database" could be considered a separate layer, eg. `delete_user()`, `edit_user()`, etc. could go into, say `module/database.sh`